nautobot/cloud/navigation.py

```diff
@@ -0,0 +1,62 @@
+from nautobot.core.apps import (
+    NavContext,
+    NavGrouping,
+    NavItem,
+    NavMenuAddButton,
+    NavMenuGroup,
+    NavMenuItem,
+    NavMenuTab,
+)
+
+menu_items = (
+    NavMenuTab(
+        name="Cloud",
+        weight=200,
+        groups=(
+            NavMenuGroup(
+                name="Accounts",
+                weight=100,
+                items=(
+                    NavMenuItem(
+                        link="cloud:cloudaccount_list",
+                        name="Cloud Accounts",
+                        weight=100,
+                        permissions=[
+                            "cloud.view_cloudaccount",
+                        ],
+                        buttons=(
+                            NavMenuAddButton(
+                                link="cloud:cloudaccount_add",
+                                permissions=[
+                                    "cloud.add_cloudaccount",
+                                ],
+                            ),
+                        ),
+                    ),
+                ),
+            ),
+        ),
+    ),
+)
+
+navigation = (
+    NavContext(
+        name="Networks",
+        groups=(
+            NavGrouping(
+                name="Cloud",
+                weight=450,
+                items=(
+                    NavItem(
+                        link="cloud:cloudaccount_list",
+                        name="Cloud Accounts",
+                        weight=100,
+                        permissions=[
+                            "cloud.view_cloudaccount",
+                        ],
+                    ),
+                ),
+            ),
+        ),
+    ),
+)
```

Comment:
This is for the react UI right? Honestly I'd rather we not add it in the first place since it doesn't do anything relevant at present, AFAIK.